exercises/reverse-string/description.md

```diff
@@ -0,0 +1,104 @@
+# Reverse String
+
+The introductory exercise to reverse an input string.
+
+This is a typical first program for beginning programming in a new language
+or environment after being able to do simple tasks.
+
+The objectives are simple:
+
+- Write a function that reverses the input string.
+- Run the test suite and make sure that it succeeds.
+- Submit your solution and check it at the website.
+
+## Tutorial
```

Comment:
You can remove everything starting with `## Tutorial` up to the end of the file, since it is specific to JavaScript.

The tutorial stuff can be added by each tracks maintainer if they feel the need.

The section about how to run the tests and how to set up the environment (as a student) are generated automatically by each track.